DiamondLightSource/SynchWeb
# LIMS-103: Fix various bugs in Screens UI

**JIRA ticket**: [LIMS-103](https://jira.diamond.ac.uk/browse/LIMS-103)

**Summary**:

Fix various minor bugs in the adding and editing of Screens.

**Changes**:
- Clicking "Add Screen" now adds a new screen to the current page rather than the next page
- The Global attribute is now set on creation
- If you are on the proposal that 'owns' a screen, you can now edit the container type (and therefore the capacity), or set it to null (default capacity of 96)
- Cell A1 is now selected for you when you first go to a screen
- Fix mis-spellings of "Crystallisation"

**To test**:
- Go to proposal mx17352, then click "Imaging" in the top right, then click "Crystallisation Screens" (under "Imaging Dashboard")
- Click "Add Screen", check an empty row appears on the page you are on
- Name the screen and set global to "Yes", click the tick, then click on the screen, check the Global has been set to Yes.
- Edit the container type and check the capacity updates. Check it perseveres after a page refresh. Set it back to null and check the capacity returns to 96, even after a page refresh.
- Go back to the Screens list, click on Ax-CP-CUSTOM-101. Check position A1 is highlighted and the components are shown.
- Go back to the Screens list, click on Test Screen 1a (on the last page). This is owned by a different proposal, check the container type is not displayed and the capacity is not editable.
